### PR TITLE
terraform: add missing policies for AWS ALB

### LINKED ISSUE #3056: Missing permission for AWS Application Load Balancer
### Issue description

The Control Plane Policy does not have this permission:
elasticloadbalancing:DescribeListenerCertificates

This is required for Ingress that create an ALB with Certicates

### Steps to reproduce the behavior

_No response_

### Version

_No response_

### Constellation Config

_No response_

### PR DESCRIPTION
**Pull request by burgerdev**
### Context

Node role permissions are currently handcrafted to allow the set of use cases we identified so far. We did not consider the use of AWS LBC as an Ingress provisioner, though, and thus never checked whether the policies are sufficient.

### Proposed change(s)

- Apply the [recommended policy](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.7/deploy/installation/#option-b-attach-iam-policies-to-nodes).
- Add a test to ensure ALB Ingresses can be created.
- Document considerations for using ALB Ingress with Constellation.

### Related issue
- Fixes #3056 

### Checklist

- [x] Run the E2E tests that are relevant to this PR's changes
  * https://github.com/edgelesssys/constellation/actions/runs/8938280204/job/24552109686 (intermediate state)
  * https://github.com/edgelesssys/constellation/actions/runs/8968562872 (current HEAD)
- [x] Update [docs](https://github.com/edgelesssys/constellation/tree/main/docs)
- [x] Add labels (e.g., for changelog category)
- [x] Is PR title adequate for changelog?
- [x] Link to Milestone

### CODE REVIEW

File: docs/docs/workflows/lb.md [hunk 1]
```diff
@@ -4,12 +4,25 @@ Constellation integrates the native load balancers of each CSP. Therefore, to ex
...
+The AWS Load Balancer Controller also provisions `Ingress` resources of class `alb`.
+AWS Application Load Balancers (ALBs) can be configured with a [`target-type`](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.7/guide/ingress/annotations/#target-type).
+The target type `ip` requires using the EKS container network solution, which makes it incompatible with Constellation.
+If a service can be exposed on a `NodePort`, the target type `instance` can be used.
+
+See [Application load balancing on Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html) for more information.
+
+:::caution
+Ingress handlers backed by AWS ALBs reside outside the Constellation cluster, so they shouldn't be handling sensitive traffic!
+:::
```
Comment (msanft) [inline docs/docs/workflows/lb.md:28]: We might want to place this above the explanation to signal the importance.

Just a nit though

**Review by msanft — COMMENTED** [review]
Policy and E2E test LGTM, but I lack the proficiency in this area to give a meaningful review. So @3u13r should probably double-check before merging.

**Review by 3u13r — APPROVED** [review]
LGTM

File: docs/docs/workflows/lb.md (hunk 1 above)
Comment (burgerdev) [reply docs/docs/workflows/lb.md:28]: I think the warning makes more sense after the terms have been introduced, so I lean towards keeping it at the end of this subsection. The paragraph should be short enough that the warning is hard to miss.

**Pull request merged**